changelogs/fragments/5629-add-prepend-hash-option-for-channel-id.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

Ah sorry, one last thing... new options are new features, so this should be `minor_changes`:
```suggestion
minor_changes:
```